bin/ci-test

```diff
@@ -10,17 +10,15 @@ function print_usage() {
 
 function handle_args() {
   export SUITE_NAME=$1
-  case $SUITE_NAME in
-    bundler1 | bundler2)
-      export MODULE=bundler
-      ;;
-    "")
-      print_usage
-      exit 1
-      ;;
-    *)
-      export MODULE=$SUITE_NAME
-  esac
+
+  # Only allow "bundler" as a valid SUITE_NAME
+  if [[ "$SUITE_NAME" != "bundler" ]]; then
+    echo "Error: Only 'bundler' is allowed as SUITE_NAME."
+    print_usage
+    exit 1
+  fi
+
+  export MODULE="bundler"
```

Comment:
Since we are removing bundler v1 from specs and replace suite name as `bundler` which is going to run specs for bundler v2 directly.